### ISSUE TITLE
Question about Yes/No field type

### ISSUE THREAD

**Issue by vigh-barnabas** [pluginsGLPI/fields#606]
I have 1000+ Computers.

If I add a new field with yes/no type and empty default value to the Computers.
For the existing rows in the block's table the new field's value will be set 0  and the select box it will show "No"  as selected when it's not yet been set by someone.
Is it possible to save the initial value as null or -1 so the select box will be empty? or the yes/no type is not intended for this?

The same problems applies to the rows not present in the block's table.

**Comment by tomolimo** [comment]
Hello @vigh-barnabas 

This is not possible with the Yes/No field, default value is always No. There is no third value like Empty for this field type.
What you need is not a Yes/No dropdown, but a Yes/No/Empty, which is then a specific dropdown that you can create with the dropdown field type.
This is what I did for my 14000+ computers,

Here is a definition of a Yes/No/Empty field named "Backup":
![image](https://user-images.githubusercontent.com/7569958/206115436-dd360fd8-992f-4ba1-bdf1-c74df9a16673.png)
And here are the values (to be found in Setup > Dropdowns menu):
![image](https://user-images.githubusercontent.com/7569958/206115605-1e9a7f92-5c18-4c33-bdc3-836eaaa36c3d.png)
And if needed, they can be localized into local languages.

Thank you,
Regards,
Tomolimo

**Comment by vigh-barnabas** [comment]
Hi @tomolimo. 
Just one follow up question. 

So I should always use a specific dropdown if I want to show Empty select box in the Front-end so the users sees if it been set or not? Even is the dropdown's value will be Yes/No same as the Yes/No field type?

Thanks.

**Comment by tomolimo** [comment]
Yes

**Comment by tomolimo** [comment]
Hello @1soproni 
The matter is not in the value, but in the fact that it will force the user to think about the question and about the answer.
Example:
The field we've added to Computer like shown above: if we let the Yes/No default value it will be "No" automatically, and the form will be validated because No is a valid answer: it means the Computer (in fact a server) has not been added to the backup plan (because for example it's a POC server, and such doesn't need to be in the backup plan). But if we don't provide No as the default value, then as Empty is not a valid answer it will force the tech to really think about the answer of the Backup field, and he may answer No or Yes (it has been added to the backup plan.
![image](https://user-images.githubusercontent.com/7569958/206388111-2ec8e217-1a41-4a86-aef2-525f397b4a90.png)

If this is not clear, I can re-explain, just ask me
Regards,
Tomolimo
